package.json

```diff
@@ -130,7 +130,7 @@
     "mjml": "^4.10.3",
     "mjml-browser": "^4.10.3",
     "mjml-react": "^2.0.0",
-    "ms": "^2.1.3",
+    "ms": "^3.0.0-canary.1",
```

Comment:
this is causing the build to break, canary is pre-release, not sure we want to use it

Reply:
i think 3.0.0 also uses ESM, so same issue as node-fetch

Reply:
rolling back